docs/source/asr/speaker_diarization/configs.rst

```diff
@@ -28,9 +28,10 @@ An example Speaker Diarization dataset configuration could look like:
 .. code-block:: yaml
 
   diarizer:
-    num_speakers: 2 # for each recording
+    oracle_num_speakers: ??? # if number of speakers is known input it or pass null if not known. Accepts int or path to file containing uniq-id and num of speakers of that session
+    max_num_speakers: 8 # max number of speakers for each recording. If oracle num speakers is passed this value is ignored
     out_dir: ??? 
-    paths2audio_files: null # either list of audio file paths or file containing paths to audio files for which we need to perform diarization.
+    paths2audio_files: ??? # either list of audio file paths or file containing paths to audio files for which we need to perform diarization.
```

Comment:
Both sentences mean same thing

Reply:
yes the change is moving from `null` to `???`

Reply:
Clear up sentence language then. It's confusing v